pkg/chartutil/requirements.go

```diff
@@ -258,6 +258,24 @@ func ProcessRequirementsEnabled(c *chart.Chart, v *chart.Config) error {
 	}
 
 	var chartDependencies []*chart.Chart
+	// If any dependency is not a part of requirements.yaml
+	// then this should be added to chartDependencies.
+	// However, if the dependency is already specified in requirements.yaml
+	// we should not add it, as it would be anyways processed from requirements.yaml
+
```

Comment:
ROTFL. After writing that, I googled "Sea Slug Long Term Memory"... and discovered that they actually do memory research on sea slugs. https://www.livescience.com/17683-sea-slug-memory.html